pkg/release/schema/release/deploy/suspend/bool.go

```diff
@@ -14,6 +14,5 @@ func (b Bool) String() string {
 }
 
 func (b Bool) Verify() error {
-	// TODO
```

Comment:
We implemented `suspend` here, nothing else to verify, since this field is just a boolean.